appcli/cli_builder.py

```diff
@@ -51,10 +51,6 @@ def create_cli(configuration: Configuration, desired_environment: Dict[str, str]
         configuration (Configuration): the application's configuration settings
     """
     APP_NAME = configuration.app_name.upper()
-    ENV_VAR_CONFIG_DIR = f"{APP_NAME}_CONFIG_DIR"
-    ENV_VAR_GENERATED_CONFIG_DIR = f"{APP_NAME}_GENERATED_CONFIG_DIR"
-    ENV_VAR_DATA_DIR = f"{APP_NAME}_DATA_DIR"
-    ENV_VAR_ENVIRONMENT = f"{APP_NAME}_ENVIRONMENT"
```

Comment:
I think we've now broken a bunch of `docker-compose.yml` files in our projects which rely on these?
Are these accessible in Jinja2 templates? If not, we have no way of using the locations in our `docker-compose.yml.j2` file

Reply:
- Environment variables are now defined by the launcher.j2 again
- Moved the 'checking' of these environment variable to be consolidated into the checking function itself (no need for these to be more globally scoped)